### PR TITLE
[14.0] s_product_brand: add bind all btn + counter

### PR DESCRIPTION
**Pull request by simahawk**

### CODE REVIEW

**Review by sebastienbeau — APPROVED** [review]
LGTM (code review) thanks for the addition !

File: shopinvader_product_brand/views/shopinvader_backend_view.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    <record model="ir.ui.view" id="shopinvader_backend_form_view">
+        <field
+            name="name"
+        >shopinvader.backend.form (in shopinvader_product_brand)</field>
+        <field name="model">shopinvader.backend</field>
+        <field name="inherit_id" ref="shopinvader.shopinvader_backend_view_form" />
+        <field name="arch" type="xml">
+            <button
+                name="%(shopinvader.shopinvader_category_binding_wizard_act_window)d"
```
Comment (sebastienbeau) [inline shopinvader_product_brand/views/shopinvader_backend_view.xml:12]: Please put the bind_all brand in the develepper tab like we have done for the product and categ

**Review by sebastienbeau — CHANGES_REQUESTED** [review]

File: shopinvader_product_brand/views/shopinvader_backend_view.xml (hunk 1 above)
Comment (simahawk) [reply shopinvader_product_brand/views/shopinvader_backend_view.xml:12]: but for now we have only this action and no wizard. I'd add it to both places. What to do you think?

Comment (simahawk) [reply shopinvader_product_brand/views/shopinvader_backend_view.xml:12]: can you check again? I added recompute actions too.

Comment (sebastienbeau) [reply shopinvader_product_brand/views/shopinvader_backend_view.xml:12]: ok !

**Review by sebastienbeau — APPROVED** [review]

**Pull request merged**